src/main/java/com/solvd/carsharing/repository/EventRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.solvd.carsharing.repository;
+
+import com.solvd.carsharing.event.Event;
+import org.springframework.data.cassandra.repository.AllowFiltering;
+import org.springframework.data.cassandra.repository.ReactiveCassandraRepository;
+import reactor.core.publisher.Flux;
+
+public interface EventRepository extends ReactiveCassandraRepository<Event, String> {
+
+    @AllowFiltering
```

Comment:
it's not a good idea to query by the non-unique and non-primary fields.
especially in Cassandra

Reply:
And what is the way to get all the events of certain aggregate? Is it a good idea to make field 'car number', 6675AI as example, unique? And than change this method to findAllByNumber(String number)?

Reply:
@h1alexbel, wdyt?

Reply:
@h1alexbel unique - I mean add method existsByNumber before saving